geometry_msgs/Point[] points
uint8 another_field
float64[] Q_array
time capture_time
